pkg/test/framework/components/stackdriver/kube.go

```diff
@@ -172,8 +191,14 @@ func (c *kubeComponent) ListLogEntries() ([]*loggingpb.LogEntry, error) {
 	}
 	var ret []*loggingpb.LogEntry
 	for _, l := range r.Entries {
+		if !strings.HasSuffix(l.LogName, logNameFilter) {
+			continue
+		}
 		// Remove fields that do not need verification
 		l.Timestamp = nil
+		l.Trace = ""
+		l.SpanId = ""
+		l.LogName = ""
```

Comment:
why are we emptying this out? we should rather include this in logentry and use it in verification in json tmpl files.
This would help differentiate log entry files easily?

/cc @bianpengyuan

Reply:
It is used to filter logs in the code above. It is filtered because it has project information in it, which relies on platform so not proper for verification.